control-plane/agents/src/common/errors.rs

```diff
@@ -21,6 +21,8 @@ pub enum SvcError {
     GetNodes { source: ReplyError },
     #[snafu(display("Node '{}' is not online", node))]
     NodeNotOnline { node: NodeId },
+    #[snafu(display("Pool owner Node '{}' is not online", node))]
```

Comment:
nit: I think we put the var names in the format string everywhere, might as well do it here?
```suggestion
    #[snafu(display("Pool owner Node '{node}' is not online"))]
```